app/src/main/kotlin/com/hippo/ehviewer/ui/reader/GalleryPager.kt

```diff
@@ -41,7 +46,18 @@ fun GalleryPager(
                 reverseLayout = type == RIGHT_TO_LEFT,
                 key = { it },
             ) { index ->
-                item(items[index])
+                val page = items[index]
+                val zoomableState = rememberZoomableState(zoomSpec = zoomSpec)
+                Box(
+                    modifier = Modifier.fillMaxSize().zoomable(
+                        state = zoomableState,
```

Comment:
Content shifts if zoomed with two finger.

Reply:
Any idea how we fix this?

Reply:
Looks like it's on compose foundation side.